debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java

```diff
@@ -232,7 +232,7 @@ private void skipMessage(final Long lsn) throws SQLException, InterruptedExcepti
      * acknowledged with the replication slot, causing any ever growing WAL backlog.
      * <p>
      * This situation typically occurs if there are changes on the database server,
-     * (e.g. in a blacklisted database), but none of them is in a whitelisted table.
+     * (e.g. in a blacklisted database), but none of them is in table.include.list.
```

Comment:
```suggestion
     * (e.g. in a excluded database), but none of them is in table.include.list.
```

Reply:
very good. have to do another commit so will add it there

Reply:
Done